test/reporters/tests/verbose.test.ts

```diff
@@ -35,9 +35,9 @@ test('prints skipped tests by default', async () => {
   expect(trimReporterOutput(stdout)).toMatchInlineSnapshot(`
     "✓ fixtures/pass-and-skip-test-suites.test.ts (4 tests | 2 skipped) [...]ms
        ✓ passing test #1 [...]ms
-       ↓ skipped test #1
        ✓ passing suite (1)
          ✓ passing test #2 [...]ms
+       ↓ skipped test #1
```

Comment:
new reporter API prints all skipped tests at the bottom for consistency

but with time I find this feature weird... maybe we should rethink that

Reply:
Fix in https://github.com/vitest-dev/vitest/pull/7752